### DISCUSSION TITLE
Actix Router with Fastly Compute@Edge

### DISCUSSION THREAD

**Discussion by Abiti-936** [actix/actix-web#2835] [Q&A]
I need to build a Rest API with Fastly Compute@Edge & it's really Great, but the routing isn't really good because it's based on matching the request path with Rust built-in match. so can I use Actix Router? Also Note that Fastly Compute@Edge Compiles Rust to WebAssembly.

for a reference see the docs & their project template in here:
https://developer.fastly.com/learning/compute/rust/
https://github.com/fastly/compute-starter-kit-rust-default/

**Comment by robjtede** [comment]
I'm not sure that `actix-router` is generally usable (yet) but I can recommend `matchit` to try out.

**Reply by Abiti-936** [reply]
Ok, I will try matchit. Thanks.
